h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -219,7 +263,7 @@ private void run() throws Exception {
 
     public PrintTreeOptions(boolean setdecimalplaces, int nplaces, int fontsize, boolean internal) {
       _setDecimalPlace = setdecimalplaces;
-      _nPlaces = _setDecimalPlace?nplaces:_nPlaces;
+      _nPlaces = _setDecimalPlace ? nplaces : _nPlaces;
```

Comment:
Minor: Formatting should normally be done separately. But this makes sense and is ok to me.